src/test/java/ay2122s1_cs2103t_w16_2/btbb/storage/JsonAdaptedOrderTest.java

```diff
@@ -2,35 +2,76 @@
 
 import static ay2122s1_cs2103t_w16_2.btbb.storage.JsonAdaptedOrder.MISSING_FIELD_MESSAGE_FORMAT;
 import static ay2122s1_cs2103t_w16_2.btbb.testutil.Assert.assertThrows;
-import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_BY_BENSON;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalClients.BENSON;
+import static ay2122s1_cs2103t_w16_2.btbb.testutil.TypicalOrders.ORDER_FOR_BENSON;
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import org.junit.jupiter.api.Test;
 
 import ay2122s1_cs2103t_w16_2.btbb.exception.IllegalValueException;
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Address;
+import ay2122s1_cs2103t_w16_2.btbb.model.client.Name;
 import ay2122s1_cs2103t_w16_2.btbb.model.client.Phone;
 
 public class JsonAdaptedOrderTest {
+    private static final String INVALID_NAME = "R@chel";
```

Comment:
Could you add the `CLIENT` part to all variables thanks
```suggestion
    private static final String INVALID_CLIENT_NAME = "R@chel";
```